src/components/BSelect/BSelect.vue

```diff
@@ -69,7 +69,7 @@ export default {
       required: true
     },
     /**
-    * Array of objects {id: number, displayName: string} to be displayed as options.
+    * Array of objects {id: number, displayText: string} to be displayed as options.
```

Comment:
Were these renamed just to be consistent?

Reply:
yes